File: 1293-three-consecutive-odds/1293-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        count = 0
        for i in range(len(arr)):
            if arr[i] & 1:
                count+=1
            else:
                count = 0
            if count==3:
                return True

        return False if count!=3 else True

File: 1293-three-consecutive-odds/1293-three-consecutive-odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int c = 0;
        for(auto i: arr)
        {
            if(i%2!=0)
            {
                c++;
            }
            else{
                c=0;
            }
            if(c==3)return true;

        }
        return false;
    }
};